<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WazeefaTech</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 33, 33); color: white; font-family: &quot;Calibri&quot;;</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>WazeefaTech</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>520</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {background-color: rgba(0, 0, 0, 0); border-radius: 20px;}
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); font-size: 20pt;</string>
   </property>
   <property name="text">
    <string>Current Version is 10.0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Update is available. New Version is 10.15.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DownloadUpdate">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(255, 0, 0); color: white; font-size: 20pt; border-radius: 10px; }
QPushButton:hover { background-color: rgb(200, 0, 0); }
QPushButton:pressed { background-color: rgb(150, 0, 0); }</string>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(0, 0, 255); color: white; font-size: 12pt; border-radius: 5px; }
QPushButton:hover { background-color: rgb(0, 0, 200); }
QPushButton:pressed { background-color: rgb(0, 0, 150); }</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color:  rgb(145, 218, 0); color: white; font-size: 12pt; border-radius: 5px; }
QPushButton:hover { background-color: rgb(0, 200, 0); }
QPushButton:pressed { background-color: rgb(0, 150, 0); }</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
